debian/libfm-qt11.symbols

```diff
@@ -368,6 +368,10 @@ libfm-qt.so.11 libfm-qt11 #MINVER#
  _ZN2Fm12FolderConfig10getIntegerEPKcPi@Base 0.16.0
  _ZN2Fm12FolderConfig10setBooleanEPKcb@Base 0.16.0
  _ZN2Fm12FolderConfig10setIntegerEPKci@Base 0.16.0
+ (arch=i386 armhf)_ZN2Fm12FolderConfig13getStringListEPKcPj@Base 1.1.0
```

Comment:
I would be careful here. Compare the c++filt output between 371 and 372, same with 373 and 374. I suspect the symbols will fail once this change hits the archive. Remind me to use a CI train PPA to test.